### PR TITLE
fix: Avoid My Collection price filter crash

### PR DESCRIPTION
**Pull request by olerichter00**
The type of this PR is: **Bugfix**

This PR resolves [CX-2263]

### Description

Avoid My Collection price filter crash.

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/main/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/main/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [x] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- fix: Avoid My Collection price filter crash - ole

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

[CX-2263]: https://artsyproduct.atlassian.net/browse/CX-2263?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/lib/Scenes/MyCollection/MyCollection.tsx [hunk 1]
```diff
@@ -234,10 +234,10 @@ const MyCollection: React.FC<{
           }
 
           return filter(artworks, (a) => {
-            if (isNaN(a.pricePaid.minor)) {
+            if (isNaN(a.pricePaid?.minor)) {
               return false
             }
-            const pricePaid = a.pricePaid.minor / 100
+            const pricePaid = a.pricePaid?.minor === 0 ? 0 : a.pricePaid?.minor / 100
```
Comment (olerichter00) [inline src/lib/Scenes/MyCollection/MyCollection.tsx:240]:
```suggestion
            const pricePaid = a.pricePaid?.minor / 100
```

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (fix: Avoid My Collection price filter crash - ole)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against b12da396e850f105667d97da22f2643d916901ac
</p>

**Review by MounirDhahri — DISMISSED** [review]
LGTM! Thanks for this fix. 
Do you maybe know if we have any tests around this, it might be useful to add a regression test to avoid this in the future. If this gets too complicated, it's fine as well, it's not a show stopper.

**Comment by olerichter00** [comment]
#squashongreen

**Review by MounirDhahri — APPROVED** [review]

**Pull request merged**